packages/jest-runtime/src/should_instrument.js

```diff
@@ -35,7 +35,10 @@ export default function shouldInstrument(
     return true;
   }
 
-  if (config.testRegex && filename.match(config.testRegex)) {
+  if (
+    config.testRegex &&
+    new RegExp(replacePathSepForRegex(config.testRegex)).test(filename)
```

Comment:
Shouldn't we normalize this in normalize.js?

Reply:
yes please.

Reply:
Moved to normalize and dropped from search_source as well.